mlflow_faculty/trackingstore.py

```diff
@@ -258,6 +262,18 @@ def create_run(
                 None if parent_run_id is None else UUID(parent_run_id),
                 tags=[mlflow_tag_to_faculty_tag(tag) for tag in tags],
             )
+        except FacultyExperimentDeleted as conflict:
+            raise MlflowException(
+                # "Non active experiment with id: {}".format(
+                #     conflict.experiment_id
+                # )
+                "Experiment {} is deleted."
+                " To create runs for this experiment,"
+                " first restore it with the shell command "
+                "'mlflow experiments restore {}'".format(
+                    conflict.experiment_id, conflict.experiment_id
```

Comment:
To avoid repeating arguments to `.format()`, I recommend doing:

```python
"Experiment {0} is deleted."
" To create runs for this experiment,"
" first restore it with the shell command "
"'mlflow experiments restore {0}'".format(
    conflict.experiment_id
)
```